docs/resources/ec_deployment.md

```diff
@@ -44,17 +44,71 @@ resource "ec_deployment" "example_minimal" {
 
 The following arguments are supported:
```

Comment:
Several of these are in https://github.com/elastic/terraform-provider-ec/blob/master/docs/datasources/ec_deployment.md as well, might be good to keep them consistent? I'm not too attached to what I wrote on there, so if you think maybe we should change some of them that's fine, but I think they should both say the same thing if the field is the same :)